example_palette_usage.py

```diff
@@ -0,0 +1,51 @@
+"""
```

Comment:
I don't get what's the point of this file at the root of the library project.

You might want to make a small example and add to the documentation (e.g. docs/Unicode.md) or create tests with this example - ideally both.